harvest/cli.py

```diff
@@ -73,6 +81,8 @@ def _validate_arguments(self, args):
         parsed = urlparse(args.repo)
         if not (parsed.scheme and parsed.hostname and parsed.path):
             return "ERROR: repo url must be of the form https://hostname/org/repo"
+        if not args.branch:
+            args.branch = "master"
```

Comment:
Is this not redundant as it is already set a `default` value?

Reply:
Yeah, this isn't needed